plugins/scaffolder-backend/api-report.md

```diff
@@ -556,8 +567,15 @@ export class TaskManager implements TaskContext_2 {
   // (undocumented)
   getWorkspaceName(): Promise<string>;
   // (undocumented)
+  rehydrateWorkspace(options: {
```

Comment:
Should this also be optional?

Reply:
Yeah, it's again about how report is generated :) 
Used also optional not only in interface but also in a class which implements it to fix the api-report.md